schemas/telemetry/third-party-modules/third-party-modules.4.schema.json

```diff
@@ -1344,6 +1344,10 @@
                       "description": "The base address to which the loader mapped the module.",
                       "type": "string"
                     },
+                    "isDependent": {
+                      "description": "True if the module is included in the executable's Import Directory Table.",
+                      "type": "boolean"
```

Comment:
Just want to confirm that this will never be `null`.

Reply:
Correct.  As long as the field exists, its value is either `true` or `false` in JSON.